tests/functional/u/used/used_before_assignment_typing.py

```diff
@@ -26,7 +26,7 @@
     import types
     import zoneinfo
 elif input():
-    import calendar, bisect  # pylint: disable=multiple-imports
+    import calendar, bisect, dbm  # pylint: disable=multiple-imports
```

Comment:
I feat this is starting to get too complex to understand. Are `calendar` and `bisect` here in order to test the same imports being under both branches of TYPE_CHECKING? Or are they here to test multiple imports on the same line? Either way, why would we add `dbm` to the same line, since we want `dbm` to be something not imported above. I wonder if we can just remove `bisect` from the TYPE_CHECKING branch and let that be the regression test--what do you think?

Reply:
makes sense to me, removing `bisect` should achieve the same effect as well. thanks for the suggestion!

Reply:
Removing `bisect` should achieve the same effect. Thanks for the suggestion!